tests/cli/test_cli_routes.py

```diff
@@ -811,3 +811,26 @@ def test_task_details_by_wf_id(client_env, db_engine):
     assert tasks[0]["task_command"] == "echo 1"
     assert tasks[0]["task_name"] == "tt_test_arg-1"
     assert tasks[0]["task_status"] == "REGISTERING"
+
+
+def test_workflow_details_viz(client_env, db_engine):
```

Comment:
is this test not redundant? nitpicking since the test suite is starting to get a little bloated, especially in the CLI/viz modules.

Reply:
There isn't a test case for this route.